calcom/cal.com
# typo in /more

## What does this PR do?



Fixes #4563 


## Demo
<img width="294" alt="Screen Shot 2022-09-19 at 11 26 22 PM" src="https://user-images.githubusercontent.com/31402501/191079784-f5808193-015f-4024-9ec7-9c9c40b44e51.png">

**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?



- Go to mobile view
- Click on 'more' and see the text
